scripts/badger/process_bribes.py

```diff
@@ -43,6 +44,13 @@ def step0_1(sim=False):
         claimed = SAFE.badger.claim_bribes_votium(
             registry.eth.bribe_tokens_claimable
         )
+        # Handling cvxFXS rewards as bribes (Ref: https://github.com/Badger-Finance/badger-strategies/issues/56)
+        cvxFXS_amount = SAFE.badger.sweep_reward_token(
+            CVX_FXS
+        )
+        if cvxFXS_amount > 0:
+            claimed[CVX_FXS] = cvxFXS_amount
```

Comment:
```suggestion
        if SAFE.badger.sweep_reward_token.call(CVX_FXS) > 0:
            cvxFXS_amount = SAFE.badger.sweep_reward_token.call(
                CVX_FXS
            )
            claimed[CVX_FXS] = claimed.setdefault(CVX_FXS, 0) + cvxFXS_amount
```
- add `.call` to prevent an unnecessary tx when amount is zero
- on the off chance that `claimed` already contains the `CVX_FXS` key, add the amount to that value. if not init the key with value `0` and add the amount to that.